### ISSUE TITLE
Feature Request: Use GitHub Releases functionality

### ISSUE THREAD

**Issue by tspearconquest** [memcached/memcached#798]
Greetings!

We have a memcached install in a kubernetes cluster, and we are working to simplify some of the processes around management of apps. One of those processes is checking for upstream updates. We aim to test the latest stable code in dev and then release it to our production environment on a fairly quick cadence, and so we would like to setup our CI pipeline to check for the latest upstream version daily.

We do this for several other projects by checking the GitHub Releases API from our CI Pipelines.
For example, we are able to check the latest upstream version of Redis by having our pipeline retrieve the URL https://api.github.com/repos/redis/redis/releases/latest and check the data provided there against the version we are running.

We were hoping to do this same automated update check for Memcached but the [page](https://api.github.com/repos/memcached/memcached/releases/latest) we would have the pipeline check doesn't seem to exist.
I did some research and found that the reason this page doesn't exist is because the GitHub Releases feature is not being used.

I noticed that the [DevelopmentRepos](https://github.com/memcached/memcached/wiki/DevelopmentRepos) page mentions needing more testing of the latest releases, so I thought it important to highlight that those of us who check for updates via our pipelines will be able to get the pipeline updated with the new version details more quickly, and so this helps you with testing because we can get any issues found and reported that much more quickly.

In conclusion, we would like to request if you could check into using the GitHub Releases features in addition to tagging the new versions, so that we and others can use automation to check for upstream updates.

Thank you in advance

**Comment by dormando** [comment]
Hey,

Thanks! I'll have to give this some thought; many people poll for updates to memcached without problem... I see the request logs :) but if it's not too hard to replicate the release into github releases I can add that to the scripts. The release process for mc predates github.

It might take me a while to get to it though due to the present backlog. Sorry.

**Comment by tspearconquest** [comment]
Hi, thank you! Yes, you are right its not an issue to check against the tags, apologies for implying otherwise.
